src/controller/testController.ts

```diff
@@ -225,7 +225,7 @@ export const runTests: (request: TestRunRequest, option: IRunOption) => any = in
                         }
                         try {
                             await runner.setup();
-                            const resolvedConfiguration: DebugConfiguration = option.launchConfiguration ?? await resolveLaunchConfigurationForRunner(runner, testContext, config);
+                            const resolvedConfiguration: DebugConfiguration = mergeConfigurations(option.launchConfiguration, config) ?? await resolveLaunchConfigurationForRunner(runner, testContext, config);
```

Comment:
Overall looks good to me. Just here:

Previously, if the launch config is passed from other extension, like PDE, we choice not to touch it and directly pass it to debugger.

Here we add a 'merge' logic. Did you meet any issue with the previous implementation?